### DISCUSSION TITLE
Problem with the "Welcome World" code

### DISCUSSION THREAD

**Discussion by ias052** [jspsych/jsPsych#868] [Q&A]
Hello,
I copied and pasted the "Welcome World!" tutorial code into Visual Studio Code:

```
1<!DOCTYPE html>
2<html>
3   <head>
4        <title>My experiment</title>
5      <script src="jspsych-6.0.5/jspsych.js"></script>
6        <script src="jspsych-6.0.5/plugins/jspsych-html-keyboard-response.js"></script>
7      <link href="jspsych-6.0.5/css/jspsych.css" rel="stylesheet" type="text/css"></link>
8    </head>
9  <body></body>
10    <script>
11
12   var hello_trial = {
13      type: 'html-keyboard-response',
14        stimulus: 'Hello world!'
15   }
16
17   jsPsych.init({
18   timeline: [hello_trial]
19   })
20
21    </script>
22 </html>
```

I saved it as "experiment.html" in the root folder "Moving-Research-Online", together with the "jspsych" folder.

When I tried to open it in Chrome, the stimulus "world_trial" does not appear.

The console message is "Uncaught ReferenceError: jspsych is not defined at experiment.html:15".
Without solving this issue I cannot progress in learning jsPsych (I am taking "Moving Research Online" workshop).
I would appreciate help in resolving this problem.

**Comment by legatrix** [comment]
With the code as you have it now, the folder inside the root must have the name 'jspsych-6.0.5'. If it has a different name, like 'jspsych', it won't know where to load the scripts from. You'll have to change either the folder name or the `src` arguments in your code above accordingly. Does that help?

**Reply by ias052** [reply]
Hi Legatrix! I compared the code that I wrote following the tutorial with Josh de Leeuw with the code from the tutorial. You are right--once I renamed  the subfolder jspsych to "jspsych-6.0.5", the tutorial code works. But the code that has the lines without "-6.0.5" in the code I wrote
```
<script src="jspsych/jspsych.js"></script>
<script src="jspsych/plugins/jspsych-html-keyboard-response.js"></script>
<link href="jspsych/css/jspsych.css" rel="stylesheet">
```
still does not work (with the name "jspsych"). 
Thank you for the first suggestion! Do you have another suggestion why plain "jspsych" (without 6.0.5) does not work?

**Reply by legatrix** [reply]
Are you still getting the same console error in the latter case? Are you sure the capitalization matches up? So both the links and the folders refer to jspsych and not jsPsych? The only other thing I could think of is trying with the newer version (6.1.0). I'm no expert though.

**Reply by vijaymarupudi** [reply]
Can you check for errors in your browser console?

Firefox: https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console
Chrome: https://developers.google.com/web/tools/chrome-devtools/open#console
